cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -343,6 +360,23 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  @Test def showIndexes() {
+    executePreparationQueries {
+      List("CREATE INDEX name FOR (c:Country) ON (c.name)")
+    }
+
+    profileQuery(
+      title = "List indexes",
```

Comment:
For consistency I would prefer the title to be "Show indexes" (remember to also update the link in the execution plans table)

Reply:
I think that all show commands have their titles be list, I know the index and constraints do because I checked that when choosing title (It was list before when it was a procedure and was not updated when the section started using the command)

Reply:
Ok that is true, if you fix the operators names and the includes/rendering, this can be as it is.